### DISCUSSION TITLE
不能用了

### DISCUSSION THREAD

**Discussion by araya** [Tairraos/TikDown#15] [General]
接口备屏蔽掉了，暂时不能下载了

**Comment by Tairraos** [comment]
是的，tiktok的接口没有任何返回了。又需要找新途径了。

**Comment by Tairraos** [comment]
已经有新api了，明天重新打包

**Comment by Tairraos** [comment]
新的api需要模拟成手机ua来请求到数据，electron使用webkit内核，w3规定用户不能修改ua， 会直接报告诉错误。Refused to set unsafe header "User-Agent"
正在想解决方案。今天打不了包了。

**Reply by araya** [reply]
大佬牛逼

**Comment by Tairraos** [comment]
新版本Release了，请更新app。谢谢使用

**Comment by ghost** [comment]
似乎又无效率

**Comment by Tairraos** [comment]
![image](https://user-images.githubusercontent.com/185069/220835708-0d6b6466-cccd-4dd3-bc46-d2779556d976.png)
可以work，实测了。
